### ISSUE TITLE
Does it support custom plugins?

### ISSUE THREAD

**Issue by techhemant** [fescobar/allure-docker-service#66]

**Comment by fescobar** [comment]
@techhemant 
You could mount this directory `/allure-2.13.2/plugins` with you own plugin
Do you mean this? If not can you give me more details. Thanks.
```
allure@eb37dc536af7:/allure-2.13.2/plugins$ ls -l
total 44
drwxr-xr-x 3 root root 4096 Feb 24 18:50 behaviors-plugin
drwxr-xr-x 3 root root 4096 Feb 24 18:49 custom-logo-plugin
drwxr-xr-x 3 root root 4096 Feb 24 18:50 jira-plugin
drwxr-xr-x 2 root root 4096 Feb 24 18:49 junit-xml-plugin
drwxr-xr-x 3 root root 4096 Feb 24 18:50 packages-plugin
-rw-rw-r-- 1 root root   85 Feb 24 18:46 README.txt
drwxr-xr-x 3 root root 4096 Feb 24 18:50 screen-diff-plugin
drwxr-xr-x 2 root root 4096 Feb 24 18:50 trx-plugin
drwxr-xr-x 3 root root 4096 Feb 24 18:50 xctest-plugin
drwxr-xr-x 3 root root 4096 Feb 24 18:49 xray-plugin
drwxr-xr-x 2 root root 4096 Feb 24 18:50 xunit-xml-plugin
```

**Comment by fescobar** [comment]
@techhemant I've re-released the version `2.13.2` (remove your previous 2.13.2 locally) adding a symbolic link to allow you to mount your directory without specifying the allure version `/allure-2.13.2`, now you can use `/allure`

```
volumes:
    - ${PWD}/my-plugin:/allure/plugins/my-plugin
```

Also, I've added the documentation:
https://github.com/fescobar/allure-docker-service#allure-customized-plugins

**Comment by fescobar** [comment]
@techhemant Can I close this ticket?
